libs/ledger-live-common/src/config/defaultConfig.ts

```diff
@@ -57,6 +57,10 @@ const defaultConfig = {
         lcd: "https://full-node.mainnet-1.coreum.dev:1317",
         minGasPrice: 0.1,
       },
+      injective: {
+        lcd: "https://injective-api.polkachu.com",
+        minGasPrice: 900000000,
```

Comment:
I find the the low gas price for keplr is
higher than ours. Can you confirm that this value is enough?
https://github.com/chainapsis/keplr-wallet/blob/04f394ac30a8c5923f01dad0a4fb5ff6f8faee47/packages/extension/src/config.ts#L1918

Reply:
I took it from here : https://github.com/cosmos/chain-registry/blob/master/injective/chain.json#L23

Reply:
Any definitive answer here?
What do you think is the better source of truth between chain-registry and kepler?

Reply:
I wouldn't change it as I had failing txs with 700000000